### ISSUE TITLE
Address Book: address a11y issues found by react-axe

### ISSUE THREAD

**Issue by chadoh** [AutarkLabs/open-enterprise#1091]
This comes from #1088 

* Install `react-axe` for Address Book, as [done here for Dot Voting](https://github.com/AutarkLabs/planning-suite/pull/1085/commits/9c7245efc4929f3e9738a3120922d610baca40ab)
* Address all issues it identifies when using the app

This is similar to #1087 and #980

**Comment by ottodevs** [comment]
This issue has a bounty attached to it.
Amount: 4.00 autark
Deadline: Thu, 18 Jul 2019 22:50:08 GMT

**Comment by rkzel** [comment]
One good question is what do we do about a11y problems that we can't solve without breaking Aragon standard. For example one frequently highlighted issue is lack of sufficient contrast:
![Screenshot from 2019-12-02 22-56-32](https://user-images.githubusercontent.com/34452131/70004213-d5bc7780-1565-11ea-9b7e-44a46998a9d1.png)
This generates two warnings, one about the text and one about the link. After some adjustments I got this:
![Screenshot from 2019-12-02 22-56-54](https://user-images.githubusercontent.com/34452131/70004297-0f8d7e00-1566-11ea-96f0-1ce744e6f841.png)
No warnings. However, this is non-standard - colors are not from the theme, and they look a bit.. off. @javieralaves what do you think, please?

**Comment by javieralaves** [comment]
Closing this as I see the PR was merged and there is no way to test this on the UI.
